muikku-core-plugins/src/main/resources/META-INF/resources/swagger.yaml

```diff
@@ -5692,28 +5692,50 @@ paths:
           description: Internal server error
 
   #ME
-  /rest/me/guidanceCounselors:
+  /rest/me/dependents:
```

Comment:
Is still spelling mistake or? I've seen dependants used before? Singular form is dependant so plural is dependants?